### ISSUE TITLE
I could not install on redmine 3.2.x

### ISSUE THREAD

**Issue by archonwang** [marcodotcastro/redmine_gamification_plugin#1]
I can not install it on redmine 3.2.X

[root@localhost redmine]# bundle install
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and installing your bundle as root will break this application for all non-root users on
this machine.
Fetching gem metadata from https://gems.ruby-china.org/...........
Fetching version metadata from https://gems.ruby-china.org/...
Fetching dependency metadata from https://gems.ruby-china.org/..
Resolving dependencies....
Bundler could not find compatible versions for gem "nokogiri":
  In snapshot (Gemfile.lock):
    nokogiri (= 1.6.7.2)

  In Gemfile:
    nokogiri (>= 1.6.7.2)

    capybara was resolved to 2.6.2, which depends on
      nokogiri (>= 1.3.3)

    deface (< 1.0.0) was resolved to 0.1.0, which depends on
      nokogiri (~> 1.4.3)

    rails-html-sanitizer (>= 1.0.3) was resolved to 1.0.3, which depends on
      loofah (~> 2.0) was resolved to 2.0.3, which depends on
        nokogiri (>= 1.5.9)

    rails-dom-testing was resolved to 1.0.7, which depends on
      nokogiri (~> 1.6.0)

    roadie-rails was resolved to 1.1.1, which depends on
      roadie (~> 3.1) was resolved to 3.1.1, which depends on
        nokogiri (< 1.7.0, >= 1.5.0)

    capybara was resolved to 2.6.2, which depends on
      xpath (~> 2.0) was resolved to 2.0.0, which depends on
        nokogiri (~> 1.3)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.

**Comment by adrianobr** [comment]
Hi,
Are you using the redmine_backlogs plugin?

**Comment by archonwang** [comment]
NO,  I used other plugins but  I found thers's some comfilct with gems version each other..
